docs/knowledge_base/tutorials/understanding_gui/step_01/step_01.md

```diff
@@ -35,7 +35,9 @@ default) in the `run()` method. For example, `Gui(...).run(port=xxxx)`. Other pa
 [here](../../../../manuals/gui/configuration.md#configuring-the-gui-instance).
 
 
-Keep in mind that you have the option to format your text. Taipy uses Markdown syntax to style your
+Keep in mind that you have the option to format your text. Taipy uses different ways to create
```

Comment:
I'm really not sure it's worth repeating that here.
And if you do, make sure the order and links are the same: markdown, html, Python.